crates/gpui/Cargo.toml

```diff
@@ -96,6 +96,8 @@ objc = "0.2"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 flume = "0.11"
+open = "5.0.1"
```

Comment:
I've implemented `open_url` and `reveal_path` with the `open` crate, which is a thin wrapper over the `xdg-open <path|url>` command.

Reply:
I could instead use `ashpd::open_uri`, but it requires one to specify which of `OpenFileRequest` or `OpenDirectoryRequest` to use when opening a path because it's just a wrapper over the raw XDG service queries, while `xdg-open` just works.